examples/prophet/train.py

```diff
@@ -0,0 +1,86 @@
+# The data set used in this example is from 'https://raw.githubusercontent.com/facebook/prophet/master/examples/example_wp_log_peyton_manning.csv'
+
+import os
+import warnings
+import sys
+
+import pandas as pd
+import numpy as np
+
+import mlflow
+import mlflow.pyfunc
+
+import cloudpickle
+
+import fbprophet
+from fbprophet import Prophet
+from fbprophet.diagnostics import cross_validation
+from fbprophet.diagnostics import performance_metrics
+
+class FbProphetWrapper(mlflow.pyfunc.PythonModel):
+
+    m = None
+
+    def __init__(self, model):
+        m = model
```

Comment:
Can we use `self.model = model` instead of defining the static variable `m`?

Reply:
done, thanks!